bin/tachyon-fuse.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
```

Comment:
Just put this as a note: as the initial PR, we are happy to use bash to parse command-line argument and set up environment variables. But later on (after this adapted and used by more people), we will probably refactor this functionality into some other Java class and leave bash script a simple wrapper----the reason is it will become harder to maintain a bash script as we are adding more features / arguments in the future.

All these, for sure, are out of the scope of this PR :)